### DISCUSSION TITLE
GitHub Pro to Organization

### DISCUSSION THREAD

**Discussion by mcbreslin** [community/community#88595] [New to GitHub] [answered]
I am trying to convert my GitHub Pro to an Organization so I can create a fork on a new Private Repo.

It keeps telling me I need a Owner to continue.  Also I cannot change the name - help!

Thanks

**Comment by Shahnawazk7944** [comment, marked as answer]
When converting to an organization, you automatically become the organization owner, so there might be a technical error. Please try again later and also ensure you're logged in to the correct GitHub Pro account you want to convert.
you cannot change your username/organization name directly during the conversion. However, you can change it later after your organization is created by Danger Zone in organization Settings.

**Reply by mcbreslin** [reply]
Thank you.  I paid for a new organization and then made my Pro account a free account.  Worked fine.
